py/server/deephaven/table.py

```diff
@@ -618,13 +618,13 @@ def reverse(self) -> Table:
         except Exception as e:
             raise DHError(e, "table reverse operation failed.") from e
 
-    def sort(self, order_by: Union[str, Sequence[str]],
+    def sort(self, order_by: Union[str, Sequence[str]] = None,
              order: Union[SortDirection, Sequence[SortDirection]] = None) -> Table:
         """The sort method creates a new table where (1) rows are sorted in a smallest to largest order based on the
         order_by column(s) (2) where rows are sorted in the order defined by the order argument.
 
         Args:
-            order_by (Union[str, Sequence[str]]): the column(s) to be sorted on
+            order_by (Union[str, Sequence[str]]): the column(s) to be sorted on, default is None
```

Comment:
i dont think this can be none, you can't sort on nothing - the `order` won't make sense without an `order_by` right?